docs/source/maintenance/migration.rst

```diff
@@ -0,0 +1,44 @@
+====================
+Migration To RocksDB
+====================
+
+Iroha allows for using Postgres or Rocks Database.
+You can use the option you prefer and if you want to switch -- there is an option for you.
+**Just migrate your database!**
+Here is how
+
+.. hint:: Both migration tool and WSV checker have ``-help`` that you can call to check the available flags anytime.
+
+How to migrate
+==============
+
+After you built Iroha (or simply run it) after version Iroha 1.3 you already have the migration tools as a separate executable! 
+
+Just run the ``irohad/iroha-migrate`` with the following flags:
```

Comment:
```suggestion
Just run the ``iroha-migrate`` with the following flags:
```
Somewhere in note that by default iroha buildsystem stores binaries to `BUILD_DIR/bin/`